tests/test_reshape.py

```diff
@@ -32,13 +32,12 @@ def test_reshape():
     assert np.array_equal(ng_results, [expected_output])
 
 
-@pytest.mark.skip(reason='Needs refactoring to ngraph++')
 @pytest.mark.parametrize('axis,expected_output', [
-    (0, np.arange(120).reshape(120)),
+    (0, np.arange(120).reshape(1, 120)),
```

Comment:
Is this in line with ONNX backend tests for flatten?

Reply:
Yup, you can see it [here](https://github.com/onnx/onnx/blob/master/docs/Operators.md#attributes-17).